translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionBasic.kt

```diff
@@ -14,5 +14,5 @@ enum class DescriptionBasic(override val value: String) : StringBasedTranslatabl
     NOT_TO_BE("not to be"),
     IS("is"),
     IS_NOT("is not"),
-    HAS("has"),
+    HAS_ELEMENT("a next element"),
```

Comment:
revert this to `HAS("has")`

Reply:
Okay, got you now.
Seems like I was right in the beginning :)
Gimme a sec

Reply:
Done